engines/scumm/imuse_digi/dimuse_engine.cpp

```diff
@@ -437,6 +437,33 @@ bool IMuseDigital::isEngineDisabled() {
 	return _isEngineDisabled;
 }
 
+void IMuseDigital::stopSMUSHAudio() {
+	if (!isFTSoundEngine()) {
+		if (_vm->_game.id == GID_DIG) {
+			int foundSoundId, bufSize, criticalSize, freeSpace, paused;
+			foundSoundId = diMUSEGetNextSound(0);
+			while (foundSoundId) {
+				if (diMUSEGetParam(foundSoundId, DIMUSE_P_SND_HAS_STREAM)) {
+					diMUSEQueryStream(foundSoundId, bufSize, criticalSize, freeSpace, paused);
+
+					// Here, the disasm explicitly asks for "bufSize == 193900";
```

Comment:
preferable not put 'disasm', but rather like original imuse or something. not explicit pointing to disassembly

Reply:
Okay, I'll change that